### ISSUE TITLE
Stale data cache? ("vice-grip" and "vise-grip")

### ISSUE THREAD

**Issue by GarrettAlbright** [PokeAPI/pokeapi#680]
`move/11` has a "name" parameter of "vice-grip" (with a C) but at least one 'mon, `pokemon/98` (Krabby), has this move listed with a "name" parameter of "vise-grip" (with an S) - key path `moves[0].move.name`.

This seems to be a fairly recent change as it's broken my import script which was working yesterday IIRC.

Oddly, I can't find "vise-grip" anywhere in the code base so I'm not sure why this is happening. Maybe it was a temporary hiccup but the bad data is being cached somewhere?

**Comment by GarrettAlbright** [comment]
After further research, I've found that [this commit](https://github.com/PokeAPI/pokeapi/commit/14af88ff3a620b21c344c23ce4a66380eb5169e6#diff-13959d3219708abbbbb81b03ea85c59e98c1ee0811aff9b003465bf590dbf9a9) from way back in May changed what had previously been "vice-grip" into "vise-grip", and [this commit](https://github.com/PokeAPI/pokeapi/commit/16d18d7af048cab45932b75caffcb47ced5846b3#diff-13959d3219708abbbbb81b03ea85c59e98c1ee0811aff9b003465bf590dbf9a9) from December 27 flipped it back around again. So I'm guessing the data for the move is just being served from a stale cache.

For my own purposes the "name" of the move can be either "vice-grip" or "vise-grip" so long as it is consistent between the move data and the monster data, but I'm sure for others this will cause major issues if their own data structures are dependent on that being consistent. Maybe we should switch it back to "vise-grip" for now?

**Comment by C-Garza** [comment]
You are right, I noticed that move name changed when syncing the data. Apparently it was renamed `vise grip` in gen8. When we made our own fork of the gen8 data, we used the new name. But it looks like Veekun upstream wants to keep the old name, `vice-grip` [as seen here in this review](https://github.com/veekun/pokedex/pull/327#discussion_r589137178). I personally think it should also stay `vice-grip` as well, as it has been that name since gen1. 
Maybe you can open an issue on Veekun, as it is now the newer spelling and future games might use that spelling instead. I know that `feint attack` used to be called `faint attack` for instance, but Veekun uses the newer `feint attack`.
